modal_proto/api.proto

```diff
@@ -112,6 +112,13 @@ enum DeploymentNamespace {
   DEPLOYMENT_NAMESPACE_GLOBAL = 3;
 }
 
+// Encoding type used for user telemetry data.
```

Comment:
Maybe `TelemetryEncodingType` instead so we don't get this confused with other kinds of encodings.